config.json.example

```diff
@@ -1,6 +1,13 @@
 {
-  "token": "BOT_TOKEN",
-  "mysqldsn": "user:password@tcp(localhost:3306)/your_database_name?parseTime=true",
-  "owners": ["1234", "1234"],
-  "guildid": "1234"
+    "token": "secret",
+    "owners": ["you", "and", "your", "friends"],
+    "guildID": "some numbers",
+
```

Comment:
the StartupChannelID config option is not present

Reply:
intended